pkg/controller/controller_test.go

```diff
@@ -56,7 +57,9 @@ var _ = Describe("Controller", func() {
 		pvcInformer := cache.NewSharedIndexInformer(objSource, pvc, 0, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
 		queue := workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())
 		pvcListWatcher := k8stesting.NewFakeControllerSource()
-		controller = NewController(fakeClient, queue, pvcInformer, pvcListWatcher, importerTag)
+		var err error
+		controller, err = NewController(fakeClient, common.CDI_SELECTOR_LABEL,importerTag)
+		if error
```

Comment:
Replace this with `Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("setUpInformer: NewController error: %v", err))`